OneSignalSDK/onesignal/src/main/java/com/onesignal/OSInAppMessageRedisplayStats.java

```diff
@@ -3,31 +3,31 @@
 import org.json.JSONException;
 import org.json.JSONObject;
 
-class OSInAppMessageDisplayStats {
+class OSInAppMessageRedisplayStats {
 
     private static final String DISPLAY_LIMIT = "limit";
     private static final String DISPLAY_DELAY = "delay";
 
-    //Last IAM display time in seconds
+    // Last IAM display time in seconds
     private long lastDisplayTime = -1;
-    //Current quantity of displays
+    // Current quantity of displays
     private int displayQuantity = 0;
-    //Quantity of displays limit
-    private int displayLimit = Integer.MAX_VALUE;
-    //Delay between displays in seconds
+    // Quantity of displays limit
+    private int displayLimit = 1;
```

Comment:
Let's ensure the backend will always send this value for unlimited reshows.

Reply:
good UnitTest, on it

Reply:
are you making this change on iOS too?

Reply:
I will be but not yet

Reply:
I did a quick check on the backed, looks like redisplay won't even get sent unless this value is set to 1 is what we want.